### DISCUSSION TITLE
azure function - how to access the azure function context inside resolver?

### DISCUSSION THREAD

**Discussion by ntodorov** [apollographql/apollo-server#5304] [Q&A] [answered]
Hi,
When using apollo-server-azure-functions, I need access to the Azure function context object inside of my resolver function.
Does `apollo-server-azure-functions` provide that?
If not any workarounds?

Example:
```javascript
  const server = new ApolloServer({ typeDefs, resolvers });  

  module.exports = server.createHandler();
```
and a normal azure function (used inside createHandler() ) we have
```javascript
const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
....
}
```
so I need access to the `context: Context`.

**Comment by glasser** [comment, marked as answer]
As [documented](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#middleware-specific-context-fields) it is the `context` option passed to your `context` function (these two `context`s are different!)

```
new ApolloServer({
  typeDefs,
  resolvers,
  context: ({ context }) => ({
    azureFunctionsContext: context,
  }),
})
```

```
  someResolver(_parent, _args, { azureFunctionsContext }) {
    // look at azureFunctionsContext
  }
```

**Reply by ntodorov** [reply]
I somehow missed that documentation! Thanks for answering!
